tests/components/dlna_dmr/conftest.py

```diff
@@ -139,3 +139,8 @@ def async_get_local_ip_mock() -> Iterable[Mock]:
     ) as func:
         func.return_value = AddressFamily.AF_INET, LOCAL_IP
         yield func
+
+
+@pytest.fixture(autouse=True)
+def dlna_dmr_mock_get_source_ip(mock_get_source_ip):
+    """Mock network util's async_get_source_ip."""
```

Comment:
The `dlna_dmr` component actually uses `async_upnp_client.utils.async_get_local_ip`, not the hass version, for IPv6 support and an easier interface.

It is already mocked a few lines up.

Reply:
@chishm `dlna_dmr` lists `network` as a requirement in its manifest, `network` calls `async_get_source_ip` during its setup. If this is not needed, `network` should be removed from the requirements.

Reply:
Ah, I see it got added during the transition to `components.network.async_get_source_ip` by #52980, before I rewrote that part of `dlna_dmr`.
I can remove `network` from the manifest, but it's a transitory requirement via `ssdp`. Does that mean the fixture is still required? If it is required, would it not be better to make the top-level fixture in `tests/conftest.py` auto-use?

Reply:
If `dlna_dmr` depends on `ssdp` and `ssdp` depends on `network` the fixture is still needed.

Most integrations don't depend, directly or indirectly on `network`, so I'm not sure we want to make it a top-level autouse fixture.